internal/common/metrics.go

```diff
@@ -5,41 +5,25 @@ package common
 
 import (
 	"fmt"
-	"sync"
 	"time"
 
 	"github.com/cactus/go-statsd-client/v5/statsd"
-	"go.uber.org/multierr"
 )
 
 // StartLogWrite starts go routines to write logs to each of the logs that are monitored by CW Agent according to
```

Comment:
By `StartLogWrite`, do you mean `StartSendingMetrics`?

Reply:
I didn't add that code. But yes, I think it should have been `StartSendingMetrics`.
I think we can do a separate follow up PR to tidy up this code. For now I reverted my changes to this file since they were unrelated to the current task.